x/iro/keeper/settle.go

```diff
@@ -47,18 +47,13 @@ func (k Keeper) Settle(ctx sdk.Context, rollappId, rollappIBCDenom string) error
 		return errorsmod.Wrapf(gerrc.ErrInternal, "required: %s, available: %s", plan.TotalAllocation.String(), balance.String())
 	}
 
-	// burn all the unsold IRO token.
+	// burn all the remaining IRO token.
 	iroTokenBalance := k.BK.GetBalance(ctx, k.AK.GetModuleAddress(types.ModuleName), plan.TotalAllocation.Denom)
 	err := k.BK.BurnCoins(ctx, types.ModuleName, sdk.NewCoins(iroTokenBalance))
 	if err != nil {
 		return err
 	}
 
-	// claim the fee token
```

Comment:
we claim the tokens on creation instead
otherwise we need to handle the scenario where creationFee param changed